lib/phoenix_live_view/tokenizer.ex

```diff
@@ -642,7 +657,13 @@ defmodule Phoenix.LiveView.Tokenizer do
   end
 
   defp handle_interpolation(<<>>, _line, _column, _buffer, _state) do
-    {:error, "expected closing `}` for expression"}
+    {:error,
+     """
+     expected closing `}` for expression
+
+     In case you don't want `{` to begin a new interpolation, \
+     you may write it using `&lbrace;` or using `<%= "{" %>`\
```

Comment:
```suggestion
     you may write it using `&lbrace;` or using `{"{"}`\
```

Reply:
This won't work. We don't count `{` inside strings because that would require full Elixir parsing.